core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/AutoTuner.scala

```diff
@@ -603,6 +602,21 @@ class AutoTuner(
     recommendClassPathEntries()
   }
 
+  def getShuffleManagerClassName() : String = {
+    val shuffleManagerVersion = appInfoProvider.getSparkVersion.get.filterNot("().".toSet)
+    val finalShuffleVersion : String = if (platform.getName == PlatformNames.DATABRICKS_AWS
+      || platform.getName == PlatformNames.DATABRICKS_AZURE) {
+      val dbVersion = appInfoProvider.getProperty(
+        "spark.databricks.clusterUsageTags.sparkVersion").getOrElse("")
+      dbVersion match {
+        case ver if ver.contains("10.4") => "321db"
+        case ver if ver.contains("11.3") => "330db"
+        case _ => "332db"
```

Comment:
This is a good start. We can have this list hardcoded for this PR.
However moving forward, it is going to maintain the code everytime a new DB class is added.

Perhaps later we can create a static properties file to map between db and plugin versions? At least we won't need to change scala code for that. Also, we can automate a script whenever a new class is added in the plugin repo. WDYT?

Reply:
Completely agree. This needs to be maintained some place. Do we have examples of static properties files elsewhere at the moment? If not, we can start with this property. Automating is more interesting as adding a new class may not be a complete indicator that we need a new config key, DB in particular might fit that use case.

Reply:
I started the seed to this infrastructure in the existing PR #705 ([loadPropertiesFile](https://github.com/NVIDIA/spark-rapids-tools/pull/705/files#diff-aec92c36bb6f6d3cd355e10b7a03fd45526f92e19560912d427f759c8315b15eR91))
My plan is to build on that by adding `trait configurable` that will load the the properties file from a static file.

Reply:
Should I address the change here as part of this PR or are we good with this for now?

Reply:
This is good for this PR.